{
  "name": "failing-line",
  "version": "0.1.0",
  "description": "Return the line number and filename of failing line from given error object",
  "main": "index.js",
  "scripts": {
    "test": "node test.js"
  },
  "devDependencies": {
    "tape": "*"
  },
  "keywords": [
    "error",
    "fail",
    "failing",
    "line number"
  ],
  "repository": {
    "url": "git@github.com:azer/failing-line.git",
    "type": "git"
  },
  "author": {
    "name": "azer"
  },
  "license": "BSD",
  "bugs": {
    "url": "https://github.com/azer/failing-line/issues"
  },
  "homepage": "https://github.com/azer/failing-line",
  "_id": "failing-line@0.1.0",
  "dist": {
    "shasum": "fc14fd8f0801a641f1144e4e893010ab6fea2496",
    "tarball": "http://registry.npmjs.org/failing-line/-/failing-line-0.1.0.tgz"
  },
  "_from": "failing-line@>=0.1.0 <0.2.0",
  "_npmVersion": "1.4.3",
  "_npmUser": {
    "name": "azer",
    "email": "azer@kodfabrik.com"
  },
  "maintainers": [
    {
      "name": "azer",
      "email": "azer@kodfabrik.com"
    }
  ],
  "directories": {},
  "_shasum": "fc14fd8f0801a641f1144e4e893010ab6fea2496",
  "_resolved": "https://registry.npmjs.org/failing-line/-/failing-line-0.1.0.tgz"
}
